nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py

```diff
@@ -241,7 +242,7 @@ def __init__(
 
 class SpanishCharsTokenizer(BaseCharsTokenizer):
 
-    PUNCT_LIST = get_ipa_punctuation_list("es-ES")
+    PUNCT_LIST = get_ipa_punctuation_set("es-ES")
```

Comment:
Should this (and the following `self.punct_list`s) be updated to indicate they are sets? Alternatively, change `get_ipa_punctuation_set` back to `list` and convert to list before returning.

Reply:
I think the safest thing is to have it return a sorted list.

The original PUNCT_LIST variables are actually tuples, which is a bit confusing.